pom.xml

```diff
@@ -83,6 +83,7 @@
     <netty.boringssl.version>2.0.7.Final</netty.boringssl.version>
     <netty.version>4.1.24.Final</netty.version>
     <zookeeper.version>3.4.13</zookeeper.version>
+    <fastjson.version>1.2.58</fastjson.version>
```

Comment:
If we add new dependency, we need to update the license file [here](https://github.com/apache/servicecomb-pack/blob/master/distribution/src/release/LICENSE)